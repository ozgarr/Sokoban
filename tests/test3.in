@z+-

z2
z4
z8
z6
0
z6
z4
z6
z6
0
0
0
0
.
