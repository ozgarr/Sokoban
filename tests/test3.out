@z+-
@z+-
@z+-
@z+-
-@Z-
@z+-
-@Z-
-@Z-
--*z
--*z
-@Z-
@z+-
@z+-
@z+-
